### PR TITLE
Homework10: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**

### CODE REVIEW

**Comment by ETsiunel** [comment]
@Stml89 PR is ready for review

File: hw_10.py
Comment (Stml89) [inline hw_10.py:1]: it would be nice to have a separate file for each task

File: hw_10.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+def validate_arguments(function_to_decorate):
+    """Декоратор проверки аргументов функции"""
+    def validator_wrapper(*args):
+        function_to_decorate(*args)
+        for i in args:
+            if i <= 0:
+                raise ValueError("Argument can not be zero or negative."
+                                 f" Invalid argument: {i}")
```
Comment (Stml89) [inline hw_10.py]:
```suggestion
        for i in args:
            if i <= 0:
                raise ValueError("Argument can not be zero or negative."
                                 f" Invalid argument: {i}")
        function_to_decorate(*args)
```

Comment (Stml89) [reply hw_10.py]: Just to check arguments type before function call

**Review by Stml89 — APPROVED** [review]
Fix comments and we are good to go

File: hw_10.py
Comment (ETsiunel) [reply hw_10.py:1]: ok, will be done (starting from the next HW)

File: hw_10.py (hunk 1 above)
Comment (ETsiunel) [reply hw_10.py]: fixed

**Pull request merged**